docs/Design/Null Check.md

```diff
@@ -0,0 +1,140 @@
+# Null Check
+Creates checks for if the passed variable has value null and/or undefined (see errata). Can be used to check positvly for null or negetivly for null

```

Comment:
Since GitHub's markdown viewer puts sentences with only one endline between them on the same line, put an endline between sentences. It makes it easier for Git diffs.